backend/tests/services/kanban/Linear/Linear.test.ts

```diff
@@ -0,0 +1,158 @@
+import { expect } from "chai";
+import "mocha";
+import sinon from "sinon";
+import linearCards from "../../../fixture/LinearCards";
+import linearCardHistory from "../../../fixture/LinearCardsHistory";
+import linearWorkflowStates from "../../../fixture/LinearWorkflowStates";
+import { StoryPointsAndCycleTimeRequest } from "../../../../src/contract/kanban/KanbanStoryPointParameterVerify";
+import { Linear } from "../../../../src/services/kanban/Linear/Linear";
+import { LinearClient } from "@linear/sdk";
+import { LinearVerifyToken } from "../../../../src/services/kanban/Linear/LinearVerifyToken";
+import { KanbanTokenVerifyModel } from "../../../../src/contract/kanban/KanbanTokenVerify";
+import {
+  IssueConnection,
+  LinearFetch,
```

Comment:
same issue
and I found there still have some `import` issues in the previous code, could you please improve it BTW? 
Thx

Reply:
done